### ISSUE TITLE
sensitive argument not set to true in output.tf for "mssql" at line 82

### ISSUE THREAD

**Issue by rituraj-tiwari** [terraform-google-modules/terraform-google-sql-db#310]
### TL;DR

Hello,

While trying to run the terraform by using this module, it is throwing the error as:

Error: Output refers to sensitive values
│
│   on .terraform\modules\sql-db_mssql\modules\mssql\outputs.tf line 82:
│   82: output "primary" {
│
│ Expressions used in outputs can only refer to sensitive values if the sensitive attribute is true.

When I add "sensitive = true" for the output at line 82, it works fine. Also, at the moment I am not aware of a workaround if there is any, where I can make some changes in my terraform code that would make the configuration not ask for setting "sensitive = true"

Can someone have a look into this?

### Expected behavior

The terraform should not throw the error that I got.

### Observed behavior

It is throwing an error:

Error: Output refers to sensitive values
│
│   on .terraform\modules\sql-db_mssql\modules\mssql\outputs.tf line 82:
│   82: output "primary" {
│
│ Expressions used in outputs can only refer to sensitive values if the sensitive attribute is true.

### Terraform Configuration

```hcl
module "sql-db_mssql" {
  source  = "GoogleCloudPlatform/sql-db/google//modules/mssql"
  version = "10.1.0"
  backup_configuration = var.backup_configuration
  project_id           = var.project_id
  db_name              = var.db_name
  name                 = var.name
  random_instance_name = var.random_instance_name
  tier                 = var.tier
  database_flags       = var.database_flags
  database_version     = var.database_version
  deletion_protection  = var.deletion_protection
  region               = var.region
  zone                 = var.zone
}
```

### Terraform Version

```sh
Terraform version being used is v0.15.0
```

### Additional information

Apart from this module for "mssql", the same issue exists for "mysql" (source  = "GoogleCloudPlatform/sql-db/google//modules/mysql")

Error: Output refers to sensitive values
│
│   on .terraform\modules\sql-db_mysql\modules\mysql\outputs.tf line 117:
│  117: output "primary" {
│
│ Expressions used in outputs can only refer to sensitive values if the sensitive attribute is true.
╵
╷
│ Error: Output refers to sensitive values
│
│   on .terraform\modules\sql-db_mysql\modules\mysql\outputs.tf line 127:
│  127: output "instances" {
│
│ Expressions used in outputs can only refer to sensitive values if the sensitive attribute is true.

I was able to resolve the error by adding “sensitive = true” (as per the [article](https://www.terraform.io/language/upgrade-guides/0-14#sensitive-values-in-plan-output)) in the [outputs.tf](http://outputs.tf/) of the root module, into the output arguments on lines 117 and 127
